### PR TITLE
[MIG] invoice_discount: Migration to 14.0

### PR DESCRIPTION
**Pull request by hugho-ad**

### CODE REVIEW

**Comment by hugho-ad** [comment]
@ebirbe @moylop260 

could you review pls

**Comment by hugho-ad** [comment]
you can test it at 
~~https://git.vauxoo.com/vauxoo/lasec/-/merge_requests/316~~

https://git.vauxoo.com/vauxoo/lasec/-/merge_requests/317

File: invoice_discount/hooks.py [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+            create_column(cr, table, column, "numeric")
+
+
+def post_init_hook(cr, registry):
+    _logger.info("Compute discount columns")
+    env = Environment(cr, SUPERUSER_ID, {})
+
+    env.cr.execute("""
+        UPDATE account_move_line
+        SET subtotal_wo_discount = quantity * price_unit
```
Comment (moylop260) [inline invoice_discount/hooks.py:32]:
```suggestion
        SET subtotal_wo_discount = quantity * price_unit,
            discount_amount = discount * subtotal_wo_discount / 100
```

Comment (moylop260) [reply invoice_discount/hooks.py:32]: forget it

**Review by moylop260 — APPROVED** [review]
👍

**Comment by ebirbe** [comment]
Hello @hugho-ad this MR looks good to me. You can merge.

Selección_365

But in the [PR of lasec](https://git.vauxoo.com/vauxoo/lasec/-/merge_requests/317) there is some fixes to be done. I will put my comments there.

**Pull request merged**